tests/test_browser.py

```diff
@@ -1681,6 +1681,11 @@ def test_gles2_emulation(self):
                        '-s', 'FULL_ES2=1', '-lGL', '-lEGL', '-lX11',
                        '--preload-file', 'basemap.tga', '--preload-file', 'lightmap.tga', '--preload-file', 'smoke.tga'])
 
+  @requires_graphics_hardware
+  def test_clientside_vertex_arrays_es3(self):
+    # NOTE: Should FULL_ES3=1 imply client-side vertex arrays? The emulation needs FULL_ES2=1 for now.
```

Comment:
I agree with this comment - we should turn on `FULL_ES2` when `FULL_ES3` is set. Can do that in emcc.py. This could be left to another PR though, but if so we should file an issue so we don't forget it, unless you think you can get to in very soon.

Reply:
In fact I do not mean to turn on `FULL_ES2` when `FULL_ES3` is set (I'm neutral to this), and the documentation says [these two flags are orthogonal](http://kripken.github.io/emscripten-site/docs/porting/multimedia_and_graphics/OpenGL-support.html#opengl-es-2-0-3-0-emulation). I'm just wondering if client-side vertex arrays should be enabled here. It's a feature supported by ES2, but discouraged in ES3.

Reply:
Thanks, good point, I didn't realize that about those flags.

Reply:
(Not sure about the answer to the question about client-side arrays.)